docs/llb/llb-auth-api.md

```diff
@@ -388,6 +392,7 @@ Body:
     "diagnosis": STRING OR NULL,
     "medication": STRING OR NULL,
     "notes": STRING OR NULL,
+    "profilePicture": IMAGE OR NULL,
```

Comment:
I believe everything in the response shouldn't be an IMAGE it should be a STRING